view/frontend/layout/customer_account.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceBlock name="customer_account_navigation">
+            <block class="Magento\Customer\Block\Account\SortLinkInterface" name="customer-account-navigation-my-credit-cards-link">
+                <arguments>
+                    <argument name="path" xsi:type="string">omise/methods/cards</argument>
```

Comment:
why do we need `methods` in the URL here.. 🤔

Reply:
as omise payment methods... I might change?